### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Adanzeakonobi**
In this branch  added the following; 
- index.html that contains body part
- style.css that contains position, media query and color.
- .gitignore
- image folders

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -488,3 +510,289 @@ textarea {
...
+  color: #fff;
+}
+
+.text-black {
+  color: #1c1a19;
+}
+
+.font-crete-round {
+  font-family: "Crete Round", serif;
+}
```
Comment (gemmen29) [inline css/style.css:861]: - [ ] Kindly make sure that you're using the same font family that is used in the Figma template. You're actually using the correct family font. However, you're not importing it. This font doesn't exist in the browser by default which is why the browser is using the fallback font provided which is "serif" in your case. To import the font you have to add it from Google Fonts from [this link](https://fonts.google.com/specimen/Crete+Round) and add it to your website either through CSS imports or using `<link>` meta tags in your HTML.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
         </a>
       </nav>
     </header>
-    <header class="header-box">
```
Comment (gemmen29) [inline index.html:20]: - [ ] Kindly adjust the spacing around the nav elements to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
...
+    <section class="about-us">
       <section class="header-content">
         <h1 class="header-title">Hey There. I'm Adaobi</h1>
         <h2 class="header-subtitle">I'm a Software Developer</h2>
```
Comment (gemmen29) [inline index.html:33]: - Kindly try to stick as much as possible to the Figma design, which includes:
   - [ ] Aligning this section's elements to match the Figma design.
   - [ ] Adjusting the spacing between the background image and the text so they don't overlap.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -58,13 +58,19 @@ <h2 class="header-subtitle">I'm a Software Developer</h2>
...
         </ul>
       </section>
-    </header>
+    </section>
     <main>
-      <section class="new-container">
+      <header>
         <div class="work-container">
-          <h6 class="recent-work">My Recent Works</h6>
-          <hr class="about-hr" />
```
Comment (gemmen29) [inline index.html:66]: - [ ] Kindly adjust the spacing around this heading text to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 5]
```diff
@@ -170,13 +162,12 @@ <h6 class="art-print">Profesional Art Printing Data</h6>
...
           </div>
-          <ul class="pro-tag">
-            <li class="pro-h">html</li>
-            <li class="pro-h">bootstrap</li>
-            <li class="pro-h">Ruby</li>
-          </ul>
-          <button class="see-proj" type="button">See Project</button>
         </div>
       </section>
       <section class="about">
```
Comment (gemmen29) [inline index.html:183]: - Kindly try to stick as much as possible to the Figma design, which includes:
   - [ ] Aligning this section's elements to match the Figma design.
   - [ ] Adjusting the spacing between the background image and the text so they don't overlap.
   - [ ] Adjusting the width of the horizontal line to match the Figma design.

| Your design | Figma design | 
| --- | --- |
| image | image |

File: index.html [hunk 6]
```diff
@@ -192,8 +183,10 @@ <h2 class="about-head">About me</h2>
...
         </div>
-        <hr class="about-separater" />
-        <div>
+        <div class="about1">
+          <hr class="about-separater" />
+        </div>
+        <div class="footer-technologies">
           <ul class="technologies">
             <li class="tech-title font-inter">
               <img src="./Skeleton/diamond.png" alt="diamond" />
```
Comment (gemmen29) [inline index.html:202]: - [ ] Please try to align this section to be more in the center and spread across the view width, like in the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 7]
```diff
@@ -228,36 +221,67 @@ <h2 class="about-head">About me</h2>
...
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+            <p>
+              <button class="button" type="submit">Get in touch</button>
+            </p>
+          </form>
```
Comment (gemmen29) [inline index.html:266]: - [ ] Kindly increase the width of the form elements to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 8]
```diff
@@ -228,36 +221,67 @@ <h2 class="about-head">About me</h2>
...
+            <img class="social-icon" src="./icons/twitter.png" alt="Twitter" />
+          </a>
+          <a href="#" class="social-link">
+            <img class="social-icon" src="./icons/medium.png" alt="Medium" />
+          </a>
+        </div>
+        <div class="footer-line-section">
+          <img src="assets/png/footer-line-shape.png" alt="Footer Line Shape" />
+        </div>
+      </footer>
```
Comment (gemmen29) [inline index.html]: - Kindly try to stick as much as possible to the Figma design, which includes:
   - [ ] Adding the horizontal line above the social media to match the Figma design
   - [ ] Adjusting the color of the social media icons to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Adanzeakonobi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
- Please keep in mind that the Figma design desktop design is made in width of `1440px`. However, the requirement of this project says to build the desktop design for your project on `768px`. Using fixed widths from the design will result in a good-looking website on width 1440px but anything lower than that to 768px, the responsivity would be ruined. Please keep that in mind and use percentage width instead of fixed widths.
- [ ] _Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (Adanzeakonobi) [reply index.html:266]: @gemmen29 Please how am i supposed to maintain the same width if you ask i use a template of 768px as against the figma template of 1440px?

File: index.html (hunk 5 above)
Comment (Adanzeakonobi) [reply index.html:183]: @gemmen29 Please how am i supposed to maintain the same width if you ask i use a template of 768px as against the figma template of 1440px?

File: index.html (hunk 3 above)
Comment (Adanzeakonobi) [reply index.html:33]: @gemmen29 Please how am i supposed to maintain the same width if you ask i use a template of 768px as against the figma template of 1440px?

**Review by gemmen29 — COMMENTED** [review]
@Adanzeakonobi You choose a media query of 768px, so the design is supposed to be similar to the Figma design in that width you choose. You can achieve this by not using the fixed width as the Figma design and instead of using width with percentage.

**Review by gemmen29 — COMMENTED** [review]
@Adanzeakonobi You don't need to use the fixed width in the Figma design exactly. The design is for guideness. You need your design to be responsive in all widths and this is not achievable with the fixed width.

File: index.html (hunk 7 above)
Comment (Deepakdanger) [reply index.html:266]: @Adanzeakonobi, By using `%` in style you can build a more responsive website.

File: index.html [hunk 9]
```diff
@@ -228,36 +221,64 @@ <h2 class="about-head">About me</h2>
           </ul>
         </div>
       </section>
-  <section class="contact">
-  <form action="https://formspree.io/f/xlezlkyw" method="post">
-          <p class="form-detail">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-             <input type="text" placeholder="Full Name" maxlength="30" required>
-             <input type="email" placeholder="Email address" required>
-             <textarea name="msgt1" id="t1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+      <section class="contact">
```
Comment (Deepakdanger) [inline index.html:234]: - Kindly try to stick as much as possible to the Figma design as requested by previous reviewer, which includes:-
- [ ] The width of the input should be full, use % in style to make it responsive.
- [ ] There is a border line above icons, Please include it.
- [ ] The color of the icons should be dark, please change the color of the icon.
- Your design

Screenshot from 2022-03-27 02-11-48

- Figma design
Screenshot from 2022-03-27 02-11-17

File: index.html (hunk 2 above)
Comment (Deepakdanger) [inline index.html:20]: - [ ] As requested by the previous reviewer, the navbar includes the 4 option in the **desktop** version, and adjust the spacing between them.
 
- Figma design 
Screenshot from 2022-03-27 02-12-29

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Adanzeakonobi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
Hi @Adanzeakonobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**